.github/workflows/pylint.yml

```diff
@@ -0,0 +1,69 @@
+name: Linting
+on:
+  pull_request:
+    branches:
+      - '*'
+permissions:
+  pull-requests: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
```

Comment:
Is there a need to use python 3.9? pylint doesn't seem to yet work with 3.12, but 3.11 works fine

Reply:
I think we can change it to 3.11